Python/Product/PythonTools/PythonTools/OutliningTaggerProvider.cs

```diff
@@ -323,7 +322,7 @@ class OutliningWalker : PythonWalker {
             }
 
             public override bool Walk(ClassDefinition node) {
-                AddTagIfNecessary(node, node.HeaderIndex, node.Decorators);
+                AddTagIfNecessary(node, node.HeaderIndex + 1, node.Decorators);

```

Comment:
Yes.  FunctionDef it works fine.   For ClassDef, Headerindex points to the end of the class def and not the ':'  so if we collapse we also collapse the ':' into the '...' that displays for the collapsed region.  There is definitely a non zero chance the header index is set incorrectly on classdef and I am compensating here.  If you want I could make a bug to chase that down and remove this + 1.